packetinjector/pcap.go

```diff
@@ -43,33 +44,52 @@ func (p *PcapPacketGenerator) PacketSource() chan *Packet {
 	go func() {
 		defer close(ch)
 
-		for {
-			data, _, err := p.pcapReader.ReadPacketData()
+		var pcapReader *pcapgo.Reader
+		var err error
+
+		for i := uint64(0); i < p.Count; i++ {
```

Comment:
It would be nice if we could have a way to say : use all the packets instead of having to specify the exact Count